python/tvm/autotvm/task/dispatcher.py

```diff
@@ -60,6 +60,21 @@ def query(self, target, workload):
             ret = self._old_ctx.query(target, workload)
         return ret
 
+    def update(self, target, workload, cfg):
+        """
+        Update context with a specific config.
```

Comment:
This is a quite critical design here, and I think it is best if we could provide some motivation(with an example) on why do we need to expose this API)

Reply:
We can add a Note block to this function